Twig/Extension/FormatterMediaExtension.php

```diff
@@ -109,4 +109,14 @@ public function path($media, $format)
     {
         return $this->getTwigExtension()->path($media, $format);
     }
+
+    /**
+     * NEXT_MAJOR: Remove this method when bumping requirements to twig 2.
+     *
+     * {@inheritdoc}
+     */
+    public function getName()
```

Comment:
Not sure if I understand you..

When this PR (https://github.com/sonata-project/SonataFormatterBundle/pull/231) gets on a release of FormatterBundle we will not need those changes.